color[] colors
---
structArray points_profi
structArray points_drill
string error
